src/include/daos/mem.h

```diff
@@ -398,18 +398,20 @@ umem_has_tx(struct umem_instance *umm)
 	return umm->umm_ops->mo_tx_add != NULL;
 }
 
-#define umem_alloc_verb(umm, flags, size)				\
-({									\
-	umem_off_t	__umoff;					\
-									\
-	__umoff = (umm)->umm_ops->mo_tx_alloc(umm, size, flags,		\
-					      UMEM_TYPE_ANY);		\
-	D_ASSERTF(umem_off2flags(__umoff) == 0,				\
-		  "Invalid assumption about allocnot using flag bits");	\
-	D_DEBUG(DB_MEM, "allocate %s umoff "UMOFF_PF" size %zu\n",	\
-		(umm)->umm_name, UMOFF_P(__umoff), (size_t)(size));	\
-	__umoff;							\
-})
+#define umem_alloc_verb(umm, flags, size)                                                          \
+	({                                                                                         \
+		umem_off_t __umoff;                                                                \
+                                                                                                   \
```

Comment:
```suggestion
												   \
```

Reply:
ah, I see what is happening here.   clang-format fills in the end of line with spaces but checkpatch requires a blank line and expects tabs at start of line.   I think this can be ignored.   clang-format is going to overtake checkpatch eventually.